libmscore/rendermidi.cpp

```diff
@@ -1693,6 +1693,8 @@ int articulationExcursion(Note *noteL, Note *noteR, int deltastep)
             if (!e || e->type() != ElementType::CHORD)
                   continue;
             Chord* chord = toChord(e);
+            if (chord->vStaffIdx() != chordL->vStaffIdx())
```

Comment:
Originally I was comparing the `staffMove()` properties, which worked because currently we know that both chords will have the same `staffIdx()`. But it think it is worth the extra computation to write it this way. Perhaps we should also be considering chords that "belong" to a different staff but appear in this one. If so, then we definitely want to compare the `vStaffIdx()` values.